zk_toolbox/crates/zk_supervisor/src/commands/test/build.rs

```diff
@@ -0,0 +1,61 @@
+use common::{cmd::Cmd, spinner::Spinner};
+use config::EcosystemConfig;
+use xshell::{cmd, Shell};
+
+use crate::messages::{
+    MSG_INTEGRATION_TESTS_BUILDING_CONTRACTS, MSG_INTEGRATION_TESTS_BUILDING_DEPENDENCIES,
+};
+
+const TS_INTEGRATION_PATH: &str = "core/tests/ts-integration";
+const CONTRACTS_TEST_DATA_PATH: &str = "etc/contracts-test-data";
+
+pub fn run(shell: &Shell) -> anyhow::Result<()> {
+    let ecosystem_config = EcosystemConfig::from_file(shell)?;
+
+    build_repository(shell, &ecosystem_config)?;
+    build_test_contracts(shell, &ecosystem_config)?;
+    install_and_build_dependencies(shell, &ecosystem_config)?;
+
+    Ok(())
+}
+
+fn build_repository(shell: &Shell, ecosystem_config: &EcosystemConfig) -> anyhow::Result<()> {
```

Comment:
what does it mean, build repository?  i was assuming rust buid?

Reply:
It comes from this: https://github.com/matter-labs/zksync-era/blob/553d307217282b18c2c3d7cc6f340f529bb4ade2/zk_toolbox/crates/zk_supervisor/src/commands/test/integration.rs#L43-L52

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/8fcf801dc69ffe2294ce0e4ae9aa3b1f78f2d4cb